beacon-chain/blockchain/receive_block.go

```diff
@@ -30,9 +30,9 @@ type SlashingReceiver interface {
 	ReceiveAttesterSlashing(ctx context.Context, slashings *ethpb.AttesterSlashing)
 }
 
-// ReceiveBlock is a function that defines the the operations (minus pubsub)
-// that are performed on blocks that is received from regular sync service. The operations consists of:
```

Comment:
I removed the `from regular sync service` part because we also call this method when proposing blocks.